### ISSUE TITLE
How to enable linter?

### ISSUE THREAD

**Issue by erik-f** [julia-vscode/julia-vscode#1105]
I just installed the extension and things like running code and autocomplete work just fine.

However no errors are getting highlighted.
How do I enable the linter? Unfortunately the wiki isn't helpful at all.

**Comment by davidanthoff** [comment]
The linter is enabled by default, but it is not able to find all errors in your code: it is a static linter, but Julia is a dynamic language, so there is only so much that we can find. So most likely you just don't have any errors in your code that the linter is able to catch.

**Comment by erik-f** [comment]
I don't see any error detections, even if I have errors like using undefined variables in my code.
Even in the example code from the JuliaCon 2018
```
if true
    println("TRUE")
else
    println("FALSE")
end
```
it doesn't detect that the else branch is can never be reached.

**Comment by davidanthoff** [comment]
Hm, then lets reopen this.

Are you getting hover help?

**Comment by erik-f** [comment]
Yes, hover help works just fine.

**Comment by erik-f** [comment]
I tried removing all VSCode config folders to start with a clean config and without any extensions. So it seems like it's neither depending on my config nor an extension conflict.

I also installed the plugin on my other system running Archlinux. Again the linter doesn't work, but running code and hover help work just fine.

Another thing I noticed is that autocompletion doesn't show any documentation.

**Comment by ZacLN** [comment]
Is this due to the recent change where linting is disabled on non-workspace files?

**Comment by erik-f** [comment]
Yes, you're right. Adding the folder to my workspace solves the problem, linting now works as expected. 
This should probably be added to the documentation to prevent confusion.

I still don't see any documentation in the autocompletion suggestions list though. Do I need to enable it somewhere?

Thank you both very much for the great support.

**Comment by ZacLN** [comment]
There should an ⓘ icon when you hover on the completion list that will turn documentation on

**Comment by erik-f** [comment]
Great, thank you.

**Comment by martinmestre** [comment]
Hello @ZacLN @davidanthoff @efaulhaber ,
could you tell me how I have to do in order to see the error highlighting? I am not seeing any. 
I made a workspace but still can not see any highlithing, ever for something like:
function test(x))))))
Thank you very much!

**Comment by jkcoxson** [comment]
Just wanna bump this issue... It's been 2 years and as a new Julia convert this was frustrating to debug.
If it can't be fixed, there should at least be documentation about it somewhere.

**Comment by leandropaolo1** [comment]
I still dont understand how to enable the Julia linter. Can someone help me?

**Comment by davidanthoff** [comment]
Closing as outdated: all `julia.lint.*` settings were removed in v1.220.0. Linting is on by default and is configured through a project-local `JuliaLint.toml`, so the setting this question is about no longer exists.

If the current configuration does not let you do what you need, please open a new issue describing that.
